src/server/pki.rs

```diff
@@ -3,213 +3,281 @@
 
 //! Some cryptographic utilities
```

Comment:
I guess a more meaningful summary might be in order, maybe "Routines for en-/decode custom X509 extensions"?

Reply:
reworded